common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -5637,7 +5637,10 @@ public static enum ConfVars {
 
     HIVE_ADDITIONAL_CONFIG_FILES("hive.additional.config.files", "",
             "The names of additional config files, such as ldap-site.xml," +
-                    "tez-site.xml, etc in comma separated list.");
+                    "tez-site.xml, etc in comma separated list."),
```

Comment:
can you add this a bit closer to another hive.tez.* options instead of the end of list?

Reply:
Sure. done